src/gui/dialogs/gamestate_inspector.cpp

```diff
@@ -128,7 +129,7 @@ class gamestate_inspector::model
 			pages.emplace_back(start, data.size() - start);
 		}
 	}
-	static const unsigned int max_inspect_win_len = 20000;
+	unsigned int page_characters = 10000 / preferences::pixel_scale();
```

Comment:
Needs to be `video::get_pixel_scale()`, as the `preferences` can also be set to automatic scaling.

Reply:
Updated.